extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/ImgurTransferExtension.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.datatransfer.imgur;
+
+import java.io.IOException;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import org.datatransferproject.api.launcher.ExtensionContext;
+import org.datatransferproject.api.launcher.Monitor;
+import org.datatransferproject.spi.cloud.storage.AppCredentialStore;
+import org.datatransferproject.spi.transfer.extension.TransferExtension;
+import org.datatransferproject.spi.transfer.provider.Exporter;
+import org.datatransferproject.spi.transfer.provider.Importer;
+import org.datatransferproject.datatransfer.imgur.photos.ImgurPhotosExporter;
+import org.datatransferproject.datatransfer.imgur.photos.ImgurPhotosImporter;
+import org.datatransferproject.types.transfer.auth.AppCredentials;
+
+/** Extension for transferring Imgur data */
+public class ImgurTransferExtension implements TransferExtension {
+  private static final String SERVICE_ID = "Imgur";
+  private static final String BASE_URL = "https://api.imgur.com/3";
```

Comment:
is BASE_URL used elsewhere other than passed to the exporter, if not lets move it into the Exporter

Reply:
It will be used in importer, so I left it here, but made `public static` for using in the exporter